web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -463,15 +463,15 @@
           catch (e) {
             settings.logError(e);
           }
-        };
+        }
 
-        if (evt.done != null) {
+        if (evt.done !== null) {
           doneMsg();
         }
-        else if (evt.success != null) {
+        else if (evt.success !== null) {
           successMsg(evt.success);
         }
-        else if (evt.failure != null) {
+        else if (evt.failure !== null) {
```

Comment:
We sure these three don't break anything? `undefined != null` is false while `undefined !== null` is true…